spinalcordtoolbox/reports/qc.py

```diff
@@ -75,9 +75,14 @@ def template(self, mask, ax):
         ax.get_xaxis().set_visible(False)
         ax.get_yaxis().set_visible(False)
 
-    def no_seg_seg(self, mask, ax):
+    def no_seg_seg(self, mask, ax, cmap='gray', norm=None, colorbar=False):
         """Create figure with image overlay. Notably used by sct_registration_to_template"""
-        ax.imshow(mask, cmap='gray', interpolation=self.interpolation, aspect=self.aspect_mask)
+        fig_ax = ax.imshow(mask, cmap=cmap, norm=norm, interpolation=self.interpolation, aspect=self.aspect_mask)
+        if colorbar:
+            cax = ax.inset_axes([1.005, 0.07, 0.011, 0.86])
+            cbar = mpl_plt.colorbar(fig_ax, cax=cax, orientation='vertical', pad=0.01, shrink=0.5, aspect=1, ticks=[norm.vmin, norm.vmax])
+            cbar.ax.tick_params(labelsize=5, length=2, pad=1.7)
+            ax.text(1.5, 6, '2', color='white', size=3.25)
```

Comment:
TODO: change the number 2 as an argument to have 1 and 2

Reply:
done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4602/commits/106bff867a6b4c145a6c51c8b0be2f5c9abecadb